docs/platforms/unreal/installation/installation-recommendations.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Installation Recommendations
+description: "Learn more about choosing the right installation for Unreal Engine."
+sidebar_order: 8000
+---
+
+## Installation Recommendations: GitHub Releases or Fab
+
+<Note>
+While Unreal Engine uses "plugins", we will refer to the Sentry support we offer within Unreal Engine as the Sentry Unreal Engine SDK.
+</Note>
+
+There are three common ways to install an SDK to use with Unreal Engine:
+1. Install from the [Epic Games Fab](https://www.fab.com/) ([formerly known as the marketplace](https://www.unrealengine.com/en-US/blog/fab-epics-new-unified-content-marketplace-launches-today))
+2. Download a pre-built SDK that you install (e.g. from a [GitHub Releases page](https://github.com/getsentry/sentry-unreal/releases))
+3. Clone and build the SDK yourself and install
+
+Being [Fair Source](https://fair.io/), Sentry offers all three for you to be able to get the insights only Sentry can provide. However, each has it's own limitations.
+
+### Overview of SDK Versions
+
+| Feature                    | Fab                 | GitHub Releases | Build Yourself |
+|----------------------------|---------------------|-----------------|----------------|
+| Supported engine versions  | 5.1 and newer       | 4.27 and newer  | 4.27 and newer |
+| Supported UE project types | Blueprint and C++   | C++ only        | C++ only       |
+| Backend (Windows)          | Breakpad            | Crashpad        | Crashpad       |
+| `on_crash` hook (Windows)  | Not supported       | Supported       | Supported      |
+| Sentry CLI                 | Manual download     | Included        | Included       |
+
+### Installing from Fab
```

Comment:
I think we should put this below the GH heading. Perhaps before we describe either we have a general recommendation. like:

"We recommend the GitHub release. Main advantages are: Crash reports uploaded immediately (crashpad), automatic symbol upload and larger range of supported UE versions. If you have a Blueprint only project you'd need to convert it to C++"